src/components/cylc/Header.vue

```diff
@@ -37,27 +37,79 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         </g>
       </svg>
     </div>
-    <div class="c-environment-info">
-      <v-chip id="username" class="text--secondary" variant="outlined">{{ user }}</v-chip>
-      <span class="at">@</span>
-      <v-chip id="host" class="text--secondary" variant="outlined">{{ environment }}</v-chip>
+    <div
+      id="cylc-select-options"
+      class="c-environment-info w-100 d-flex flex-column align-center px-4 row-gap-3"
+    >
+      <v-defaults-provider :defaults="{
+        VCombobox: {
+          bgColor: 'white',
+          rules: [(val) => Boolean(val) || 'Required'],
+        }
+      }">
+        <v-combobox
+          class="w-100"
+          id="cylc-owner-combobox"
+          :disabled="store.state.user.user.mode !== 'multi user'"
+          label="server owner"
+          :default="owner"
+          :items="Array.from(owners)"
+          v-model="owner"
+          @keyup.enter="owners.add(owner)"
+        />
+        <v-combobox
+          class="w-100"
+          id="cylc-deployment-combobox"
+          :disabled="store.state.user.user.mode !== 'multi user'"
+          label="deployment"
+          :default="deployment"
+          :items="Array.from(deployments)"
+          v-model="deployment"
+          @keyup.enter="deployments.add(deployment)"
+        />
```

Comment:
Actually would "host" make more sense than "deployment"?

Reply:
Deployment may refer to a collection of hosts e.g. exvcylc01-17.